Bottom: 2e368d668c9873082e8de43575ca0c9acecde19d
Top:    17257c20be5adb9b5cd75f31863d2bbd883ffaa2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 20:34:53 +0000

Mark segments as not round before falling back to line-line intersection

This avoids assert failures with debugging enabeled


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 8c34ee2..197bb3b 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1290,6 +1290,8 @@ seg_in_seg_arc_arc (struct info *i, struct seg *s1, struct seg *s2)
       s1->v->radius == s2->v->radius)
     {
       printf ("Cop-out for co-circular arcs for now - they will be better handled by their line approximations. <<<<<<<<<<\n");
+      s1->v->is_round = false;
+      s2->v->is_round = false;
       return seg_in_seg_line_line (i, s1, s2);
     }
 #endif
